recipe/install_flang.bat

```diff
@@ -1,11 +1,12 @@
 cd %SRC_DIR%\build
 call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" amd64
-nmake install
+
+cmake --build . --target install
 if errorlevel 1 exit 1
 
-rm %LIBRARY_BIN%\flang.dll
-rm %LIBRARY_BIN%\flangrti.dll
-rm %LIBRARY_BIN%\ompstub.dll
+rm %LIBRARY_BIN%\libflang.dll
```

Comment:
Why this change? Is `libflang.lib` the static library and `flang.lib` the import library for the shared library?

Reply:
libflang is the dynamic library. flang is the dynamic library. Of course it can be changed, but that's how it was when I pulled out my old repository.

Reply:
> libflang is the dynamic library. flang is the dynamic library. 

I don't understand

Reply:
see images below

Reply:
oh, sorry typo!

Reply:
Previously `flang.lib` was the import library for the dynamic library, `flang.dll`, the dynamic library and `libflang.lib` was the static library. Why change this?